index.js

```diff
@@ -82,46 +93,47 @@ function serveStatic(root, options) {
       return
     }
 
-    var forwardError = !fallthrough
     var originalUrl = parseUrl.original(req)
-    var path = parseUrl(req).pathname
+    var originalPath = parseUrl(req).pathname
```

Comment:
Calling this `originalPath` is confusing, because there is an `originalUrl`, but this path is not from that. Can the name be changed?